public/javascripts/SVLabel/spec/SVLabel/modal/ModalMissionCompleteProgressBar-spec.js

```diff
@@ -0,0 +1,107 @@
+describe("ModalMissionCompleteProgressBar tests", function () {
+	var bar;
+
+	beforeEach( function () {
+		$uiModalMissionCompleteFixture = $('<div id="modal-mission-complete-complete-bar"></div>');
+		$('body').append($uiModalMissionCompleteFixture);
```

Comment:
So, I commented on the `ModalMissionCompleteMap-spec.js` as well, but I prefer not to depend on the presence of the body tag. Can you create a body or an extra div fixture?